lib/mpi/service.rb

```diff
@@ -295,7 +296,7 @@ def message_edipi(user_identity, search_type)
       MPI::Messages::FindProfileMessageEdipi.new(user_identity.edipi, search_type: search_type).to_xml
     end
 
-    def message_user_attributes(user_identity, search_type, orch_search: false)
+    def message_user_attributes(user_identity, search_type, orch_search: false) # rubocop:disable Metrics/MethodLength
```

Comment:
I don't think you re-enable this rubocop rule (unless does it auto re-enable when you disable it inline like this?)

Reply:
your guess was correct, doing it inline like this disables it just for that method